### DISCUSSION TITLE
Search across multiple columns (and accepting substrings)

### DISCUSSION THREAD

**Discussion by kevlust** [supabase/supabase#13123] [Questions]
I have a table called `fun_facts` with the following columns:
`id`: int (primary key)
`content`: text
`keywords`: text
`tag`: text

Here's an example of a row:
`id`: 123
`content`: "A crocodile cannot stick its tongue out"
`keywords`: "impossible animals body funny"
`tag`: "fact123"

Using the Supabase Swift client, what's the **best/recommended way to search across multiple table columns**? I want the example row above to show up if I search for terms such as the following:
- "croc"
- "tongue crocodile"
- "animal funny"
- "impossible"
- "fact123"
- "123"

Thanks in advance!

**Comment by steve-chavez** [comment]
This was recently documented, see https://supabase.com/docs/guides/database/full-text-search#search-multiple-columns
